<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#mainWidget{
	background-color: #F0F0F0;
}
#slidersFrame{
	background-color: #f5f5f5;
}
#leftFrame{
	background-color: #f5f5f5;
}
QPushButton{
  background-color: #333333;
  border: none;
  border-radius: 6px;
  color: #cccccc;
  font-size: 14px;
  font-weight: 500;
  line-height: 20px;
  list-style: none;
  padding: 4px 12px;
  height:40px;
}

QPushButton:hover{
	background-color: rgba(51, 51, 51, 0.5);
 	text-decoration: none;
}
QPushButton:focus{
 	outline: 1px transparent;
}
QLabel{
	font-weight: bold;
	font-size: 16px;
}
QGroupBox{
	font-weight: bold;
	font-size: 16px;
	padding:30px 0px 0px 0px;
	border:none;
}
QLineEdit{
	border:none;
	padding:4px 10px;
	border-radius: 3px;
}
QListWidget{
	border:none;
	border-radius: 3px;
}
QSlider::groove:horizontal {
border: 1px solid #bbb;
background: white;
height: 10px;
border-radius: 4px;
}

QSlider::sub-page:horizontal {
background: qlineargradient(x1: 0, y1: 0,    x2: 0, y2: 1,
    stop: 0 #66e, stop: 1 #bbf);
background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1,
    stop: 0 #bbf, stop: 1 #55f);
border: 1px solid #777;
height: 10px;
border-radius: 4px;
}

QSlider::add-page:horizontal {
background: #fff;
border: 1px solid #777;
height: 10px;
border-radius: 4px;
}

QSlider::handle:horizontal {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #eee, stop:1 #ccc);
border: 1px solid #777;
width: 13px;
margin-top: -2px;
margin-bottom: -2px;
border-radius: 4px;
}

QSlider::handle:horizontal:hover {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #fff, stop:1 #ddd);
border: 1px solid #444;
border-radius: 4px;
}

QSlider::sub-page:horizontal:disabled {
background: #bbb;
border-color: #999;
}

QSlider::add-page:horizontal:disabled {
background: #eee;
border-color: #999;
}

QSlider::handle:horizontal:disabled {
background: #eee;
border: 1px solid #aaa;
border-radius: 4px;
}

QScrollArea{
border: none;
}


</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="leftFrame">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="browseBtn">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>Browse Files</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>150</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QGroupBox" name="orignalSignalGB_3">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>100</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QFrame{
	background-color: #F0F0F0;
	border-radius: 22px;
	border: 1px solid rgb(51, 51, 51);
}
QLabel{
	border:none;
	font-size: 12px;
}
QPushButton{
  background-color: #333333;
  border: none;
  border-radius: 12px;
  color: #cccccc;
  font-size: 14px;
  font-weight: 500;
  line-height: 19px;
  list-style: none;
  padding: 4px 4px;

}</string>
              </property>
              <property name="title">
               <string>Orignal Signal</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_19">
               <item>
                <widget class="QFrame" name="originalFrame">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>45</height>
                  </size>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_12">
                  <item>
                   <widget class="QPushButton" name="originalPlayBtn">
                    <property name="minimumSize">
                     <size>
                      <width>30</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>30</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>p</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="updatedSignalGB_3">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>100</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QFrame{
	background-color: #F0F0F0;
	border-radius: 22px;
	border: 1px solid rgb(51, 51, 51);
}
QLabel{
	border:none;
	font-size: 12px;
}
QPushButton{
  background-color: #333333;
  border: none;
  border-radius: 12px;
  color: #cccccc;
  font-size: 14px;
  font-weight: 500;
  line-height: 19px;
  list-style: none;
  padding: 4px 4px;

}</string>
              </property>
              <property name="title">
               <string>Updated Signal</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_20">
               <item>
                <widget class="QFrame" name="updatedFrame">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>45</height>
                  </size>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_13">
                  <item>
                   <widget class="QPushButton" name="updatedPlayBtn">
                    <property name="maximumSize">
                     <size>
                      <width>30</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>p</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>120</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_25">
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>Select Mode</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="selectModeComBox">
              <item>
               <property name="text">
                <string>Uniform</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Select Window</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="selectWindowComBox"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="applyChangesBtn">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>Apply Changes</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_21">
         <item>
          <widget class="QFrame" name="frame_5">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>80</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_22">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="ControllerFrame">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>60</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_15">
               <item>
                <widget class="QPushButton" name="playPauseBtn">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../../First-Task/UI/icons/play.svg</normaloff>../../First-Task/UI/icons/play.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="resetBtn">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../../First-Task/UI/icons/reset.svg</normaloff>../../First-Task/UI/icons/reset.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="uploadBtn">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../../First-Task/UI/icons/uploadFile.svg</normaloff>../../First-Task/UI/icons/uploadFile.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="zoomInBtn">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../../First-Task/UI/icons/zoomIn.svg</normaloff>../../First-Task/UI/icons/zoomIn.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="zoomOutBtn">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../../First-Task/UI/icons/zoomOut.svg</normaloff>../../First-Task/UI/icons/zoomOut.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="snapshotBtn">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../../First-Task/UI/icons/snapshot.svg</normaloff>../../First-Task/UI/icons/snapshot.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>170</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_8">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_16">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_23">
              <item>
               <widget class="QFrame" name="originalSignalGraph">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="mutatedSignalGraph">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QFrame" name="frame_9">
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_24">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QFrame" name="originalSignalSpectrogram">
                 <property name="maximumSize">
                  <size>
                   <width>250</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="mutatedSignalSpectrogram">
                 <property name="maximumSize">
                  <size>
                   <width>250</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="bottomFrame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>250</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>250</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QFrame" name="sliders">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="freqGraph">
         <property name="maximumSize">
          <size>
           <width>600</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
